pages/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-multi-comp */
```

Comment:
I'm not a fan of this eslint react rule. What do you think of disabling it globally? Creating small components can be useful, creating a new file is an overhead. Should each function be into a different file, probably not. We don't enfoce it for functions. Why enforcing it for React components?

Reply:
Fully agree. It's not always clear when you should create an extra file. You could probably come up with some heuristic (LOC, number of instance fields or hooks etc) but what would that actually achieve?

I think this is one of those rules that you never explicitly follow until it rejects some code and then you disable it. Not useful at all IMO.